src/App.js

```diff
@@ -47,7 +47,7 @@ function App() {
 
       if (route.length === 1) {
         navigate('/game')
-      }  
+      }
     })
   }, [fbBase, navigate, route.length])
```

Comment:
Add *WithAuthorization props when inserting the HOC component (*WithAuthorization)

p.ex:
```javascript
<Route path="/route" element={<SuspenseComponent lazyComponent={<RoutesManagement from="/route" to="/" />} />} />

```